### DISCUSSION TITLE
Problema con Home Assistant Companion

### DISCUSSION THREAD

**Discussion by faustozago** [indomus/forum#2660] [HUB: Home Assistant]
Ciao a tutti,
ho HASSIO installato su rasp 3B+ che lavora senza problemi , e H.A. Companion installato su uno smartphone con android 13 , e anche in questo non ho mai avuto problemi ne' di installazione ne di funzionamento IN LOCALE.
Ora sto cercando di installare H.A. Companion in un tablet LENOVO, con android 8 e non riesco ad installarlo.

Quando avvio l'app , mi riconosce in automatico una  istanza H.A. , ma l'indirizzo IP non è proprio corretto:
L'indirizzo che riconosce l'app è http://192.160.x.x:8123 , mentre l'indirizzo corretto con il quale mi collego dai browser (quindi quello del raspberry) è http://192.168.x.xxx:8123.
Se mi collego con l'indirizzo che trova lui , dopo pochi secondi ma da un messaggio di errore, tipo indirizzo non corretto .
Se mi collego con l'indirizzo del raspberry , sembra che il tablet si pianti con la schermata bianca, e non succede niente, poi devo annullare l'operazione.

Specifico che se invece di aprire l'app , accedo da browser (firefox) , e inserisco l'indirizzo del raspberry http://192.168.x.xxx:8123 ,  si apre la pagina di HA e funziona tutto correttamente.

Avete idea di quale possa essere il problema?
Grazie

**Comment by Krasto82** [comment]
Ciao, può essere che il tablet ha qualche problema con l'app avendo una vecchia versione android. Se provi su un altro telefono o tablet in locale funziona?

**Comment by 50494554524F** [comment]
a me capita a causa della plancia di default che ci mette una vita a caricare a causa del fatto che ho 13k entità... 
se ti trovassi anche te in questa situazione prova a cliccare l'icona del menu - devi essere veloce, sennò chiudi l'app e riprova fino a quando ci riesci - e poi vai nell'account utente, qui cambia la plancia di default

**Comment by faustozago** [comment]
RISOLTO , il problema era il tablet.
Ho cercato di installare l'applicazione TADO ma all'autenticazione  mi diceva che la versione del browser era troppo vecchia.
Ho aggiornato firefox, poi si è installata sia l'app TADO che l'app HA companion senza problemi.
